style.css

```diff
@@ -11,6 +11,58 @@ ul li {
   display: none;
 }
 
+.nav3 {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  height: 100vh;
+  width: 100%;
+  position: absolute;
+  opacity: 0.8;
+}
+
+nav img {
```

Comment:
- [ ] When I clicked on the navbar the mobile menu does not disappear, kindly fix it